### PR TITLE
WIP: New Globus documentation

### PR DESCRIPTION
**Pull request by jefscheepers**
Extended/improved the existing globus documentation (more handholding), updated the screenshots and in general made it readier for the new version of Globus which we are using.

### CODE REVIEW

**Review by ingridbr — APPROVED** [review]
In the sharing section we need to add how to share data from the Globus Personal endpoints. 
For being able to do it they need first to apply to be promoted to Globus Plus users.
I suggest we add this later on and we publish already what we already have

File: source/globus/cli.rst [hunk 1]
```diff
@@ -1,94 +1,95 @@
...
 Getting started with the CLI
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The `Globus CLI documentation`_. contains an adequate introduction to the CLI.
-The following only includes VSC-specific examples and tips.
+----------------------------
 
-The latest release can be locally installed in e.g. a Python Virtual
-Environment as follows::
+The Globus CLI can be installed as a python package.
```
Comment (smoors) [inline source/globus/cli.rst:14]: could you also install it as a module and add it to the docs?

there are already easyconfigs for it in the EB repo, and it's already installed in Hydra:
```
module load Globus-CLI
```

Comment (jefscheepers) [reply source/globus/cli.rst:14]: Thanks for the suggestion, I wasn't aware!

We will merge the documentation for now (we need it online today for a user) and install the module either Friday or beginning or January (depending on some other tasks).

File: source/globus/endpoints.rst [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
++------------------------------------------+-------------------+----------------+------------------+
+| VSC KU Leuven data directories           | VSC account       | No             | No               |
++------------------------------------------+-------------------+----------------+------------------+
+
+If you are one of the pilot users of the Tier-1 Data platform, you can find its collection by searching for 'vsc irods.hpc'.  
+
+.. _globus-local-endpoints:
+
+Local endpoints
+------------------
```
Comment (tvandenberghe) [inline source/globus/endpoints.rst:50]: Dashes go beyond title

File: source/globus/cli.rst (hunk 1 above)
Comment (jefscheepers) [reply source/globus/cli.rst]: Mentioned the CLI module on Hydra now

File: source/globus/endpoints.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+
+Creating a managed endpoint
+---------------------------
+
+The five Flemish universities have a `High Assurance subscription <https://www.globus.org/subscriptions>`_ for Globus, which allows the creation of extra managed endpoints. 
+
+VSC users and researchers from those universities can create their own managed endpoint under this subcription.  
+
+You can find the technical prerequites `here <https://docs.globus.org/globus-connect-server/v5.4/#globus_connect_server_prerequisites>`_.
```
Comment (tvandenberghe) [inline source/globus/endpoints.rst:77]: prerequites>prerequisites

File: source/globus/groups.rst [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+Creating a new group
+-------------------
+
+To create a new group, click on the 'Create new group' button in the upper right corner. 
+You will be asked for a group name, description, and whether the group should be publicly visible or not. 
+
+.. image:: groups/create_group.png
+
+After you click on 'Create group', you will be transported to the page of your group.
```
Comment (tvandenberghe) [inline source/globus/groups.rst:28]: transported>forwarded

File: source/globus/groups.rst [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+| Member        | Yes             | No                            | No                     | No                              |
++---------------+-----------------+-------------------------------+------------------------+---------------------------------+
+| Manager       | Yes             | Yes                           | No                     | No                              |
++---------------+-----------------+-------------------------------+------------------------+---------------------------------+
+| Administrator | Yes             | Yes                           | Yes                    | Yes                             | 
++---------------+-----------------+-------------------------------+------------------------+---------------------------------+
+
+
+.. warning:: 
+   If you promote someone to administrator, they will also be able to delete your group (see under).
```
Comment (tvandenberghe) [inline source/globus/groups.rst:71]: see under>see lower

File: source/globus/groups.rst [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+You can click on the row of anyone in this list to view more detailed information, and to modify their settings.
+
+For users who have not yet accepted your invitation:
+
+- You can resend or cancel their invitation on the bottom of the page.
+
+For users who have already accepted your invitation:
+
+- You can change their status (active or suspended), or remove them from the group.
+- You can change their role
```
Comment (tvandenberghe) [inline source/globus/groups.rst:55]: final . at end of sentence

File: source/globus/using_globus_via_web.rst [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+- In the panel collection B, browse to the place where the corresponding folder is or needs to be created.
+- In the dropdown 'Transfer & Timer Options', click the checkbox 'sync - only transfer new or changed files'.
+- In that same dropdown, scroll down and set 'Schedule Start' to the moment you want the synchronizations to start.
+- Underneath, select how often the synchronization should happen. 
+   - For example, to schedule this synchronization every week forever, set 'Repeat' to 'days', set 'every X days' to 7 and 'ends' to 'never'.
+- Click on the 'Start' button above the source collection. 
+
+.. image:: using_globus_via_web/timer.png
+
+You can get an overview of all scheduled transfers and synchronizations on 'activity' tab (on the left side of the page) under the subtab 'timers.
```
Comment (tvandenberghe) [inline source/globus/using_globus_via_web.rst:141]: on 'activity tab'>on the 'activity tab' 
'timers>'timers'

**Review by tvandenberghe — APPROVED** [review]
Reviewed typos

File: source/globus/cli.rst (hunk 1 above)
Comment (smoors) [reply source/globus/cli.rst]: thanks!

**Pull request merged**